.github/release-drafter.yml

```diff
@@ -8,14 +8,13 @@ categories:
   - title: '🐛 Bug Fixes'
     labels:
       - 'kind/bug'
-      - 'bug'
       - 'fix'
-  - title: '🛠️ Misc. Improvements'
-    label: 'maintenance'
   - title: '📖 Documentation'
     labels:
       - 'docs'
       - 'kind/documentation'
+  - title: '🛠️ Misc. Improvements'
```

Comment:
I think I did not us using other than "Other changes" from the last ones? Should we keep them? do you want to add any other?

Reply:
we didn't because the PRs were not labelled. In the past we were using them and worked quite well.